install/0000_30_machine-api-operator_08_rbac.yaml

```diff
@@ -1,13 +1,78 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: cluster-api-manager-role
+rules:
+  - apiGroups:
+      - cluster.k8s.io
+    resources:
+      - clusters
+      - clusters/status
+      - machines
+      - machines/status
+      - machinesets
+      - machinesets/status
+      - machinedeployments
+      - machinedeployments/status
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+
+  - apiGroups:
+      - machine.openshift.io
+    resources:
+      - clusters
+      - clusters/status
+      - machines
+      - machines/status
+      - machinesets
+      - machinesets/status
+      - machinedeployments
+      - machinedeployments/status
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+
+  - apiGroups:
+      - healthchecking.openshift.io
+    resources:
+      - '*'
+    verbs:
+      - '*'
 ---
+apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: default-account-openshift-machine-api
-subjects:
-- kind: ServiceAccount
-  name: default
-  namespace: openshift-machine-api
+  name: cluster-api-manager-rolebinding
 roleRef:
-  kind: ClusterRole
-  name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-api-manager-role
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
Can we create a dedicated service account? It's not much more trouble, and it means random pods started in the namespace don't get elevated permissions.